plugin-api/MetaflowPluginAPI.js

```diff
@@ -1,37 +1,37 @@
 const VERSION_INFO = {
-  api: '0.13.0'
-}
+  api: '0.13.3',
```

Comment:
As this is a breaking change, shouldn't we bump it more than a patch version or two?

Reply:
Upped